src/main/java/eu/europeana/metis/sandbox/service/util/ZipServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package eu.europeana.metis.sandbox.service.util;
 
+import eu.europeana.metis.harvesting.HarvesterException;
+import eu.europeana.metis.harvesting.http.CompressedFileExtension;
+import eu.europeana.metis.harvesting.http.HttpHarvester;
+import eu.europeana.metis.harvesting.http.HttpHarvesterImpl;
 import eu.europeana.metis.sandbox.common.exception.InvalidZipFileException;
 import eu.europeana.metis.utils.ZipFileReader;
-import java.io.ByteArrayInputStream;
-import java.io.IOException;
-import java.util.List;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.io.IOUtils;
 import org.springframework.stereotype.Service;
 import org.springframework.web.multipart.MultipartFile;
 
+import java.io.*;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
```

Comment:
If we are going the way you've implemented it in the `DatasetController`, that means this class and it's interface are not used anymore and can now be removed?

Reply:
it's been removed